components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesHistoryStorageTest.kt

```diff
@@ -845,9 +842,7 @@ class PlacesHistoryStorageTest {
         try {
             history.importFromFennec(path)
             fail("Expected v0 database to be unsupported")
-        } catch (e: PlacesException) {
-            // This is a little brittle, but the places library doesn't have a proper error type for this.
-            assertEquals("Unexpected error: Can not import from database version 0", e.message)
```

Comment:
Making this PR made me realize that we no-longer get proper error messages because of a change to how errors are handled in the UDL.  This isn't completely terrible, because the app-services code now handles reporting errors itself and those errors will have the correct messages.  However, I think we still want good strings in case the error gets printed to a log so I'm planning on fixing this one.  Tell me if you don't think that's going to be useful though.

Reply:
I just made a UniFFI release that should fix this.  The error messages aren't exactly the same as before, but they will include all data from the fields.  I think this one will read something like `PlacesApiException$UnexpectedPlacesException: reason=Unexpected error: Can not import from database version 0`.  Not exactly how I would format it, but I think it's good enough for a log file.  We just need to make sure we take the new UniFFI release in app-services before merging this one.